### ISSUE TITLE
No clear way to close navigation using VoiceOver on iPhone

### ISSUE THREAD

**Issue by bjankord** [cerner/terra-framework#273]
# Bug Report

## Description
When using VoiceOver on iPhone, there is no clear way how one would close the side navigation when navigating to a page.

## Steps to Reproduce

1. Go to terra-ui.com
2. Select Components
3. Select Abstract modal
4. User is unable to close navigation

## Expected Behavior
User has clear way to deactivate / close the navigation on mobile device.

## Environment

* Component Name and Version: 
* Browser Name and Version:
* Node/npm Version: [e.g. Node 8/npm 5]
* Webpack Version:
* Operating System and version (desktop or mobile):

**Comment by bjankord** [comment]
Resolved, had to double click on Abstract modal nav item while its selected.
